build.go

```diff
@@ -429,8 +468,6 @@ func translate() {
 func transifex() {
 	os.Chdir("gui/default/assets/lang")
 	runPrint("go", "run", "../../../../script/transifexdl.go")
```

Comment:
I think that convention still stands here? It explicitly chdirs into a subfolder from the root (well, the cwd, which is assumed to be the root), then has a relative path back to the root.